tests/braynsTestData.cpp

```diff
@@ -119,6 +120,24 @@ BOOST_AUTO_TEST_CASE(render_circuit_with_color_and_compare)
     camera.setTarget(rotCenter);
     camera.setPosition(camPos + 0.9 * (rotCenter - camPos));
 
+    auto& scene = brayns.getEngine().getScene();
```

Comment:
that outlines a bit of the problem of this approach: the user now cannot 'easily' change the shading of the entire scene (in the UI not all anymore). Also, renderers which don't evaluate the material shading mode (like the basic renderer), shade anyway the way they want :)